example/test.py

```diff
@@ -77,25 +104,49 @@ def test_expected_processes(self, web_only_container):
             if len(ps_data) == 5:
                 break
             time.sleep(delay)
-            ps_data = list_container_processes(web_only_container)
+            ps_data = filter_ldconfig_process(
+                list_container_processes(web_only_container))
 
-        assert_tini_pid_1(ps_data.pop(0), 'mysite.wsgi:application')
+        assert_that(ps_data, HasLength(5))
```

Comment:
Maybe we should remove things from the list as we assert on them and then assert that the list is empty at the end?